backend/src/main.rs

```diff
@@ -49,6 +50,11 @@ fn main() {
 
     create_db();
```

Comment:
Why don't we make create_db() into a migration?

Reply:
...because I wasn't thinking about it right and made it more complicated than it needed to be :p

Nothing stops this from being a migration.